crates/owl-recorder/Cargo.toml

```diff
@@ -41,3 +41,4 @@ dirs = "6.0.0"
 
 constants = { path = "../constants" }
 rodio = "0.21.1"
+winapi = "0.3.9"
```

Comment:
`winapi` was obsoleted by `windows`; it should be possible to make the code work using the latter